.github/workflows/PrBot.yml

```diff
@@ -0,0 +1,60 @@
+name: Bot scan
+on:
+  pull_request:
+    types: [ opened, reopened ]
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  Snyk_scanning:
+    permissions: write-all
+    name: Snyk Bot scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Snyk Bot scan
+        continue-on-error: true
+        run: |
+         echo "----------Download Latest Snyk CLI-----------"
+             npm i -g snyk@latest
+             snyk --version
+             npm install --legacy-peer-deps
+             echo "----------Authenticating Snyk-----------"
+             snyk auth ${{ secrets.SNYK_TOKEN }}
+             echo "----------Snyk Scanning-----------"
+             #snyk test --all-projects -d
+             snyk test --all-projects --strict-out-of-sync=false --yarn-workspaces --detection-depth=5 -d
```

Comment:
This command does not work with Nx monorepo afaik, and it's definitely incorrect here